style.css

```diff
@@ -409,3 +409,99 @@ h1 {
   padding-top: 24px;
   padding-bottom: 24px;
 }
+
+/* contact section */
+.contact {
+  width: 375px;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  padding-left: 24px;
+}
+
+.newProjects {
+  width: 327px;
+  color: #172b4d;
+  font-family: 'Inter' 'Gill Sans', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  margin-top: 50px;
+}
+
+form {
+  margin-top: 50px;
+}
+
+input[type=text],
+input[type=email] {
+  width: 327px;
+  padding: 12px;
+  margin-bottom: 12px;
+  display: inline-block;
+  border: none;
+  border-bottom: 1px solid #ccc;
+}
+
+textarea {
+  background-color: #fbf8f7;
+  border: none;
+  width: 327px;
+  padding: 12px;
+  font-style: italic;
+}
+
+.btn-submitContainer {
+  display: flex;
+  justify-content: center;
+  margin-top: 24px;
+}
+
+button[type=submit] {
+  width: 131px;
+  height: 48px;
+  background-color: #ff6b00;
+  border: none;
+  color: #fff;
+  padding: 12px;
+  font-weight: 400;
+  font-size: 15px;
+  margin-top: 12px;
+  cursor: pointer;
+}
+
+button[type=submit]:active {
+  background-color: #e05304;
+}
+
+button[type=submit]:hover {
+  background-color: #fff5e1;
+  border: #f55800;
+  color: #f55800;
+}
+
+button[type=submit]:disabled {
+  background-color: #f4f1f0;
+  border: none;
+  color: #5b5957;
+}
+
+footer {
+  width: 375px;
+  height: 50px;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 48px;
+}
+
+.socialmedia2 {
+  width: 200px;
+  height: 40px;
+  display: flex;
+  margin-top: 24px;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Kindly align the social media icons to the center of the page. This would make your design look identical to that of the Figma template.

### Your Design

![Screenshot_6](https://user-images.githubusercontent.com/77106962/228263571-e07c8fa7-1031-4edd-a8c1-44eac5e998e1.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/228263901-7a0a2376-ecae-4a0e-83b3-0a7e08e84bd4.png)

#### Hint: After you remove the `width` declaration from the `footer` element, you can replace the `justify-content` property with the flexbox `gap` property. 👍